From 91dce87e8d9588ed09a74e214dc5f1b70dd931e8 Mon Sep 17 00:00:00 2001
From: Eric Ernst <eric.ernst@intel.com>
Date: Sun, 31 Aug 2014 10:46:58 -0700
Subject: [PATCH 1/1] cm32181: remove noisy, uneccessary  messages

There's a printk at level KERN_ERR printed in non-error
case in the cm32181 driver.  This is a noisy message, and
is not helpful.  Remove.

Signed-off-by: Eric Ernst <eric.ernst@intel.com>
---
 drivers/iio/light/cm32181.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/iio/light/cm32181.c b/drivers/iio/light/cm32181.c
index 126d6d1d3c9e..89924b3691b6 100644
--- a/drivers/iio/light/cm32181.c
+++ b/drivers/iio/light/cm32181.c
@@ -213,8 +213,6 @@ static int cm32181_get_lux(struct cm32181_chip *cm32181)
 	if (ret < 0)
 		return ret;
 
-	printk(KERN_ERR "ARJAN: cm32181_get_lux returns %i  / %x \n", ret,ret);
-
 	lux *= ret;
 	lux *= cm32181->calibscale;
 	lux /= CM32181_CALIBSCALE_RESOLUTION;
-- 
1.7.9.5

